### PR TITLE
Profile page PR

### PR DESCRIPTION
**Pull request by Tjeerd2Code**
This adds the UI for the profile page, it displayes the username (now it displays username but this is easily changed when tudor finished the overall userdata acces) and a profile picture when they have it. Also, a default picture is added (this can also be changed if needed)
You can see some recent music memories on your profile and you have the logout and delete button.

### CODE REVIEW

**Comment by TPGamesNL** [comment]
In the future, I think we should move the logout and delete buttons to settings; the profile page should work for all users, not just for the user the app is logged in with

File: app/src/main/res/layout/activity_profile.xml
Comment (andreitudorp) [inline app/src/main/res/layout/activity_profile.xml:1]: We should change the top bar from the default one (the one visible in the Home Activity) to one that would also have a back button and/or a settings button.

**Review by TPGamesNL — COMMENTED** [review]
Do the images of the music memories on the profile page work for you? Cause it might be my internet, but for me they don't show

File: app/src/main/res/layout/activity_profile.xml
Comment (TPGamesNL) [reply app/src/main/res/layout/activity_profile.xml:1]: I think we should do that in a new PR, I'll open an issue

**Pull request merged**